scipy/stats/_continuous_distns.py

```diff
@@ -9994,6 +9994,12 @@ def _entropy(self, c, d):
         # the following result.
         return 0.5 * (1.0-d+c) / (1.0+d-c) + np.log(0.5 * (1.0+d-c))
 
+    def _fitstart(self, data, args=None):
+        # Arbitrary, but c=d=1 fails due to being on edge of bounds
+        if args is None:
```

Comment:
Is it possible that the user passed only one of `c` or `d` but not both? (In that case, what happens?)